Jenkinsfile

```diff
@@ -92,9 +94,15 @@ pipeline {
                 # log jenkins deployment image
                 kubectl get deployments.apps jenkins -n ${NAMESPACE} -oyaml -o'jsonpath={ .spec.template.spec.containers[0].image }'
 
+                # Patch Jenkins Deployment to add nexus registry pull secret
+                kubectl patch deployment jenkins -n ${NAMESPACE} --patch "\$(cat templates/jenkins-master-deployment-patch.yaml)"
```

Comment:
Can't you use `--filename` instaed of `--patch` with `$(cat...)`?

Reply:
Hi, 
I think filename is for the ressource you want to update.

`  -f, --filename=[]: Filename, directory, or URL to files identifying the resource to update
`

Reply:
Indeed, my bad!

Reply:
no problem